.github/CODEOWNERS

```diff
@@ -160,12 +160,12 @@ pkg/winperfcounters/                                     @open-telemetry/collect
 
 processor/attributesprocessor/                           @open-telemetry/collector-contrib-approvers @boostchicken
 processor/cumulativetodeltaprocessor/                    @open-telemetry/collector-contrib-approvers @TylerHelmuth
-processor/deltatocumulativeprocessor/                    @open-telemetry/collector-contrib-approvers @sh0rez @RichieSams
+processor/deltatocumulativeprocessor/                    @open-telemetry/collector-contrib-approvers @sh0rez @RichieSams @jpkrohling @djaglowski
```

Comment:
Although I volunteered to sponsor, I was unaware that another implementation had already been proposed, accepted, and begun development. I feel my involvement was therefore very limited and would prefer not to be listed here.

Reply:
Thanks for clarifying! _Removed_